Shipr-Helm/templates/shipr-frontend-deployment.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: shipr-frontend
+  labels:
+    app: shipr-frontend
+spec:
+  replicas: {{ .Values.frontend.replicas }}
+  selector:
+    matchLabels:
+      app: shipr-frontend
+  template:
+    metadata:
+      labels:
+        app: shipr-frontend
+    spec:
+      containers:
+      - name: shipr-frontend
+        image: {{ .Values.frontend.image }}
+        ports:
+        - containerPort: 80
```

Comment:
hard coded port?

Reply:
I just changed 1 variable for testing, otherwise you have to give every variable from values.yml file.

Reply:
Please write helm charts using best practices, you can take help from the documentation. The point of this project was not for you to show us that you can template a helm chart. It was for you to showcase your best practices and understanding of how to create industry standard helm charts.

Have a look here: https://helm.sh/docs/chart_best_practices/